src/test-reader/test-transformer.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+export const setupTransformHook: () => VoidFunction = require("../bundle").setupTransformHook;
+export const TRANSFORM_EXTENSIONS: string[] = require("../bundle").TRANSFORM_EXTENSIONS;
```

Comment:
Here I use `require` in order to ts not understand that folder `bundle` should be builded. If I use `import` tsc build it even if exclude it in tsconfig

Reply:
I suggest adding this comment to the code as well.

To me it seems like typescript's [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) would allow the use of imports here, but don't include those files in build. But probably not worth spending time on this right now, maybe just leave a todo in the comment.